### DISCUSSION TITLE
Dunst does not follow keyboard when switching between empty workspaces in i3wm.

### DISCUSSION THREAD

**Discussion by Samueru-sama** [dunst-project/dunst#1244] [Q&A]
I have 3 displays and I setup dunst with follow = keyboard. 

I'm using dunst for visual feedback when switching workspaces in i3. Everytime that I switch between workspaces i3 sends this: 

exec --no-startup-id notify-send -t 500 "Workspace #"

Which makes dunst show up a notification of the current workspace for half a second. 

However I don't have windows in the workspaces, dunst diplays the notifications on the primary display instead of the display with focus.

**Comment by bynect** [comment]
so it works only if there are windows? I think this has to do with the fact that unlike the mouse pointer the keyboard needs something to latch onto, but I am not too sure about that

**Reply by Samueru-sama** [reply]
I ended up "fixing it" by making i3 wrap the cursor to the display with focus, and that way it works even when the display is empty. Took me a while to get used to that change but it works now.

edit: also dunst is only following mouse as well when I made that change.
